### PR TITLE
Logs to winston only when key specified and new pokemap implementation

### PR DESCRIPTION
**Pull request by chris-j-major**

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1 +1 @@
-worker: npm start
\ No newline at end of file
+worker: node index.js
```
Comment (kmoe) [inline Procfile]: this needs to be `npm start` due to the `touch .data`

File: index.js [hunk 2]
```diff
@@ -7,15 +7,22 @@ var _ = require('lodash');
...
+    subdomain: process.env.LOGGLY_SUBDOMAIN,
+    tags: ["Winston-NodeJS"],
+    json: true
+  });
+}else{
+  winston = {
+    log   : function(type,msg){ console.log(type+"\t: "+msg); },
+    error : function(msg){ console.log("E\t: "+msg); }
+  }
+}
```
Comment (kmoe) [inline index.js:25]: nice

**Comment by kmoe** [comment]
Thanks, this is a much cleaner and easier to read implementation of `pokeMap`. If you could just revert the change to the `Procfile` I can merge this as is.

**Comment by chris-j-major** [comment]
Procfile reverted - sorry about that

**Pull request merged**